charts/nitro/templates/validator/servicemonitor.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if and $.Values.validator.enabled $.Values.validator.statefulset.metrics.enabled }}
+{{- if and $.Values.validator.enabled $.Values.validator.statefulset.serviceMonitor.enabled }}
```

Comment:
Can't find the `.Values.validator.statefulset.serviceMonitor.enabled` key,  should this be `.Values.validator.serviceMonitor.enabled` ?

Reply:
yes it should